### DISCUSSION TITLE
What is DstNetPrefix ::/195 ?

### DISCUSSION THREAD

**Discussion by broonu** [akvorado/akvorado#1388] [Q&A]
Hello,

I have some traffic showing DstNetPrefix = ::/195, with no out interface.

`SrcAddr	SrcPort	DstNetPrefix	DstAddr 	DstPort`
`X.Y.Z.26	41516	::/195		A.B.C.45	53/domain`

Looking in my router, I see the network has no route (actually is a blackhole):

```
-------------------------------------------------------------------------------
A.B.C.0/24                               Blackh* BGP VPN   01d04h49m  5
       Local VRF                                                    1
-------------------------------------------------------------------------------
```

I also got this in flows:
`{"TimeReceived":1725599173,"SamplingRate":10000,"ExporterAddress":"::ffff:F.F.F.230","InIf":10,"OutIf":0,"SrcVlan":0,"DstVlan":0,"SrcAddr":"::ffff:G.H.I.100","DstAddr":"::ffff:A.B.C.45","NextHop":"::ffff:0.0.0.0","SrcAS":16509,"DstAS":XXXXX,"GotASPath":false,"SrcNetMask":22,"DstNetMask":195}
{"TimeReceived":1725599180,"SamplingRate":10000,"ExporterAddress":"::ffff:F.F.F.239","InIf":12,"OutIf":0,"SrcVlan":0,"DstVlan":0,"SrcAddr":"::ffff:K.L.M.160","DstAddr":"::ffff:A.B.C.45","NextHop":"::ffff:0.0.0.0","SrcAS":13335,"DstAS":XXXXX,"GotASPath":false,"SrcNetMask":24,"DstNetMask":195}`

How can I filter all traffic to this ::/195 (blackhole) prefixes?

**Comment by vincentbernat** [comment]
Are you using Netflow/IPFIX or sFlow?

**Reply by broonu** [reply]
> Are you using Netflow/IPFIX or sFlow?

Netflow v9 in Nokia SR-1.

**Reply by vincentbernat** [reply]
Valid range for IPv4 is 0-32, there is nothing in the RFC about special values. Forwarding status should be used to indicate blackhole. Maybe raise a case with Nokia to understand which document defines the special values they use. Maybe switching to IPFIX may change the behavior here.

**Reply by broonu** [reply]
> Valid range for IPv4 is 0-32, there is nothing in the RFC about special values. Forwarding status should be used to indicate blackhole. Maybe raise a case with Nokia to understand which document defines the special values they use. Maybe switching to IPFIX may change the behavior here.

Vincent,

Changing to IPFIX is ​​just changing the version on my router or do I need to specify in config.yaml the change from netflow v9 to 10? Do I need to rebuild the schema?

**Reply by vincentbernat** [reply]
Nothing is needed. The version is auto-detected.

**Reply by altyarl579-art** [reply]
مرحبا
